jobs/forms.py

```diff
@@ -19,3 +19,18 @@ class Meta:
             'title': 'Job title'
         }
 
+
+class MeetupForm(forms.ModelForm):
+
+    class Meta:
+        model = Meetup
+        exclude = ['reviewer', 'review_status', 'reviewers_comment',
+            'ready_to_publish', 'published_date', 'created', 'expiration_date'
+        ]
+
```

Comment:
The reason I didn't use dropdown was that we don't what to simply select a company from a list of existing company/organisations. We want to have an ability to add a company as well, so we need a field that will enable that.

I guess that we can think of some sort of autocomplication mechanism later on, but propably it's not necessary as a browser will most probably suggest the user the previously used option once they start typing.

Also, is it not somehow discouraged to use 'exclude'?